lib/chirp_web/live/post_live/show.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ChirpWeb.PostLive.Show do
+  use ChirpWeb, :live_view
+
+  alias Chirp.Timeline
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+
+  @impl true
+  def handle_params(%{"id" => id}, _, socket) do
```

Comment:
@Vretamoza what is the difference between `handle_event` and `handle_params`?

Reply:
handle_event is a function that allows us to handle events that come from the client, handle_params allows us to load data once there is a LiveView connection, it is called after the mount function.